src/main/java/core/basesyntax/MyCalculator.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+public class MyCalculator implements Calculator {
+    public double calculate(double a, double b, char sing) throws MyCalculateException {
+        switch (sing) {
+            case '*': {
+                if (a * b >= Double.MAX_VALUE || a * b <= -Double.MAX_VALUE) {
+                    throw new MyCalculateException("Too large or little result!");
+                }
```

Comment:
```suggestion
                if (a * b >= Double.MAX_VALUE || a * b <= -Double.MAX_VALUE) {
                    throw new MyCalculateException("Too large or little result!");
                }
```
You use the same If block in every case. You can use this one time after the switch-case block.  If you will use break operator I advise good article about new switch [expression](https://www.baeldung.com/java-switch#1-the-new-switch-expression).

Reply:
I didn't understand how I need to redo it.
But I redid the calculate method, look at it.